### PR TITLE
Bugfix/Send token

### PR DESCRIPTION
**Pull request by JoeSteven**

### CODE REVIEW

File: app/src/main/java/com/dimension/maskbook/ComposeBottomSheetDialogFragment.kt [hunk 1]
```diff
@@ -134,16 +134,16 @@ private fun SendTokenConfirmModal(
         val wallet by repository.currentWallet.observeAsState(initial = null)
         wallet?.let { wallet ->
             addressData?.let { addressData ->
+                // TODO get token with chainId in SendTokenConfirmData
```
Comment (JoeSteven) [inline app/src/main/java/com/dimension/maskbook/ComposeBottomSheetDialogFragment.kt:137]: working on another branch: bugfix/lucky_drop

File: wallet/src/androidMain/kotlin/com/dimension/maskbook/wallet/repository/WalletRepository.kt [hunk 2]
```diff
@@ -221,6 +232,58 @@ class WalletRepository(
         }
     }
 
+    private var isChainRefreshed = false
+    private suspend fun refreshChainDataWithTokens() {
+        try {
+            val chains = if (isChainRefreshed) {
```
Comment (Tlaster) [inline wallet/src/androidMain/kotlin/com/dimension/maskbook/wallet/repository/WalletRepository.kt]: Can we make this as stateless as possible? :thinking:

Comment (JoeSteven) [reply wallet/src/androidMain/kotlin/com/dimension/maskbook/wallet/repository/WalletRepository.kt]: updated

**Review by Tlaster — APPROVED** [review]
LGTM!

**Pull request merged**